app/controllers/forms/delete_confirmation_controller.rb

```diff
@@ -68,8 +68,7 @@ def delete_form(form)
 
       authorize form, :can_view_form?
       if form.destroy
-        flash[:message] = "Successfully deleted #{form.name}"
-        redirect_to success_url, status: :see_other
+        redirect_to success_url, status: :see_other, success: "Successfully deleted ‘#{form.name}‘"
```

Comment:
I think this is two left curly quotes, it should be:

```suggestion
        redirect_to success_url, status: :see_other, success: "Successfully deleted ‘#{form.name}’"
```